### PR TITLE
[WIP] added simple form,devise and changed doc file too

### PR DESCRIPTION
**Pull request by GeorgeFourikis**
installed simple_form gem and devise,configured the code and added alerts on "application.html.erb" and configured config.action_mailers on "production.rb" and "development.rb".

### CODE REVIEW

File: app/views/devise/passwords/edit.html.erb [hunk 1]
```diff
@@ -0,0 +1,19 @@
+<h2>Change your password</h2>
```
Comment (lasley) [inline app/views/devise/passwords/edit.html.erb]: IMO this should align with the button like your other pages
s/your/me

Comment (lasley) [inline app/views/devise/passwords/edit.html.erb]: Actually on second thought I think I prefer your. Maybe make the button simply say `Submit`? Something though, the half-symmetry is killing me

File: app/views/devise/passwords/new.html.erb [hunk 2]
```diff
@@ -0,0 +1,15 @@
+<h2>Forgot your password?</h2>
```
Comment (lasley) [inline app/views/devise/passwords/new.html.erb]: Your and me/my contexts change between header and button.

File: config/initializers/devise.rb [hunk 3]
```diff
@@ -0,0 +1,265 @@
...
+  # confirmation, reset password and unlock tokens in the database.
+  # Devise will use the `secret_key_base` on Rails 4+ applications as its `secret_key`
+  # by default. You can change it below and use your own secret key.
+  # config.secret_key = 'c79d24cdb36a9cef9be93b738cfce706f114e0f787da400ecfa71332febae6d28f10356db32228f8ec29e0d8b40fd5d970dd0938f445c84362bbff0ba3c124ee'
+
+  # ==> Mailer Configuration
+  # Configure the e-mail address which will be shown in Devise::Mailer,
+  # note that it will be overwritten if you use your own mailer class
+  # with default "from" parameter.
+  config.mailer_sender = 'please-change-me-at-config-initializers-devise@example.com'
```
Comment (lasley) [inline config/initializers/devise.rb]: Maybe set an email here?

File: config/initializers/simple_form.rb [hunk 4]
```diff
@@ -0,0 +1,165 @@
...
+  config.default_wrapper = :default
+
+  # Define the way to render check boxes / radio buttons with labels.
+  # Defaults to :nested for bootstrap config.
+  #   inline: input + label
+  #   nested: label > input
+  config.boolean_style = :nested
+
+  # Default class for buttons
+  config.button_class = 'btn'
```
Comment (lasley) [inline config/initializers/simple_form.rb]: Might want to also add the `btn-default` class on top of just the standard `btn`. Looks slightly better IMO

**Comment by lasley** [comment]
Great work! 

I don't see any tests or helpers here though, so there is no proof that this does indeed work. Take a look at the Devise wiki for info on controller testing w/ Rspec and Capybara - https://github.com/plataformatec/devise/wiki/How-To:-Test-with-Capybara

I would say at a minimum we need to know if a user can be created and logged in. Nice to have would be verifying that the forms work.

**Comment by Se7enB2st** [comment]
👍

**Comment by lasley** [comment]
FYI when this is done, make sure to remove the WIP tag and drop a comment letting everyone know it is ready for review again.

**Pull request closed without merging**